material.h

```diff
@@ -0,0 +1,42 @@
+/** 
+ @file      material.h
+ @brief     contains the material class
+ @author    Luke Eure
+ @date      January 6 2016
+*/
+
+#ifndef MATERIAL_H
+#define MATERIAL_H
+
+#include <iostream>
+#include <vector>
+
+class Material {
+private:
+    std::vector <double> _sigma_t;
+    std::vector <double> _sigma_f;
+    std::vector <double> _chi;
+    std::vector <double> _sigma_a;
+    std::vector <std::vector <double> > _sigma_s;
+    double _nu;
+    double _s_sum;
+    int _num_groups;
+
+public:
+   
+    Material();
+    Material(std::vector <double> &sigma_t, 
+            std::vector <std::vector <double> > &sigma_s, double nu, 
+            std::vector <double> &sigma_f, std::vector <double> &chi);
```

Comment:
I found the spot in ``monte_carlo.cpp`` that uses the constructor without arguments. I think we should scrap that constructor. Instead, you should use a reference or a pointer to refer to the ``Material``. For instance:
```
Material& cell_mat;
cell_mat = &mesh.getMaterial(cell);
```
OR
```
Material* cell_mat;
cell_mat = &mesh.getMaterial(cell);
```
Note with the second structure, you would have to use ``->`` instead of ``.`` to call the Class functions and variables. Going to one of these new approaches would also be far more efficient since you wouldn't be calling the copy operator every time.